### ISSUE TITLE
wav2vec: Unsupervised Pre-training for Speech Recognition

### ISSUE THREAD

**Issue by patrick-g-zhang** [patrick-g-zhang/papers-i-read#5]
# Abstract
- unsupervised pre-training for speech recognition by learning representations of raw audio. 
- improve acoustic model training

## related work
Contrastive predictive coding

# Model 
<p align="center"><img src="https://pbs.twimg.com/media/EdZgBfBUcAAFmHc?format=png&name=medium"/ width="400"></p>
encoder network : 1) X->Z five-layer convolutional network.    

context network: mix multiple latent representations zi . . . zi−v into a single contextualized tensor ci = g(zi . . . zi−v) for a receptive field size v.

objective:
k_th step contrastive loss:
- the objective are summation of K step contrastive loss
- negative samples from the same utterance
<p align="center"><img src="https://pbs.twimg.com/media/EdZpJ5WUcAEGFRM?format=png&name=medium"/ width="400"></p>
after training, the context network c_i replace the log mel filter bank feature for acoustic model training.

# Experiment
## data
- TIMIT training hours 3 hours
- WSJ: 81 hours 
- Librispeech
## per-training data setup
-  full 81 hours of the WSJ corpus
- an 80 hour subset of clean Librispeech
- full 960 hour Librispeech training set 
- combination of all of them

## Acoustic Models
wav2letter++: waveform to character

### Pre-training for the WSJ benchmark
![image](https://user-images.githubusercontent.com/13295454/87995429-0adfff00-cb22-11ea-82e4-75d5f800f64b.png)

### Ablations
- Table 3 shows that increasing the number of negative samples only helps up to ten samples
<p align="center"><img src="https://pbs.twimg.com/media/Ediao_0UEAYXDYm?format=png&name=medium"/ width="400"></p>
